tensorflow/tools/ci_build/builds/test_on_install.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
I consolidated this file (test_on_install.sh) and gpu_pip.sh by removing gpu_pip.sh. test_on_install.sh does what gpu_pip.sh does, and more (installing and testing). There will be some minor changes in Jenkins required.